modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectStops.java

```diff
@@ -49,13 +49,7 @@
      */
     public MBObjectStops(MBLayer layer) {
         try {
-            if (layer.getPaint() != null) {
```

Comment:
We still need these null checks, else getStops will throw an NPE id paint or layout are not present (which is perfectly valid for some layers)